### PR TITLE
Example revision of the JHtmlTabs using JHtmlElement

### PR DESCRIPTION
**Pull request by Mathewlenning**
Example of how pull request #7948 can simplify our lives. 

This is less than ideal. I'd actually prefer to just send the wrapper and let the developer add the tab panes using the JHtmlElement interface.

But even in its current form it is way better than the current JHtmlTabs implementation. 

BTW does anyone know why we are adding the empty dt/dd set? or can I remove it?

### CODE REVIEW

File: libraries/cms/html/bootstraptabs.php [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+	 * @since   1.6
+	 */
+	protected static function loadBehavior($group, $params = array())
+	{
+		static $loaded = array();
+
+		if (!array_key_exists((string) $group, $loaded))
+		{
+			// Include MooTools framework
+			JHtml::_('behavior.framework', true);
```
Comment (dgt41) [inline libraries/cms/html/bootstraptabs.php]: Is there a reason to include Mootools?

Comment (Mathewlenning) [inline libraries/cms/html/bootstraptabs.php]: I didn't change the JavaScript portion, just the rendering of the actual HTML

**Comment by dgt41** [comment]
@Mathewlenning I think you based this on the old html/tabs.php which is not the bootstrap version, thus the dd, dl elements and also the Mootools library dependency and the extra tabs.js script. The bootstrap version should be located inside the html/bootstrap.php file

**Comment by Mathewlenning** [comment]
O.k. Thanks @dgt41 

I'm glad to know that JHtmlTabs isn't actually being used! I'll make a different example after I've looked at the bootstrap file.

**Pull request closed without merging**